spec/proto/basic.spec.coffee

```diff
@@ -27,11 +27,11 @@ describe "Basic setup", ->
     expect($F(select)).toBe ""
 
     container = div.down(".chosen-container")
-    container.simulate("mousedown") # open the drop
+    Chosen.trigger(container, "mousedown") # open the drop
     expect(container.hasClassName("chosen-container-active")).toBe true
 
     #select an item
-    container.select(".active-result").last().simulate("mouseup")
+    Chosen.trigger(container.select(".active-result").last(), 'mouseup');
```

Comment:
Nitpicky, but inconsistent quote usage. On multiple locations, but marking them all feels too extreme.

And yes I know it's hypocritical, because we use single and double quotes randomly around the library, but I still strive for some consistency.